cli/js/buffer_test.ts

```diff
@@ -78,8 +78,8 @@ function repeat(c: string, bytes: number): Uint8Array {
 
 test(function bufferNewBuffer(): void {
   init();
-  const buf = new Buffer(testBytes.buffer as ArrayBuffer);
-  check(buf, testString);
+  const buf = new Buffer(testBytes!.buffer as ArrayBuffer);
```

Comment:
We should use `assert(testBytes)` here so the not null assertion isn't required.

Reply:
I only now learned about TypeScript 3.7's "assertion functions" 🙂
I'll update my PR to take advantage of that wherever possible.